client/my-sites/email/email-providers-comparison/index.jsx

```diff
@@ -642,6 +696,8 @@ export default connect(
 			selectedDomainName: ownProps.selectedDomainName,
 		} );
 
+		const domainName = ownProps.cartDomainName ?? domain.name;
```

Comment:
Does it also make sense to wire in an explicit `hasCartDomain` prop that we can guarantee will be a boolean? This will simplify a number of the code locations that need to make decisions based on the `domain` object.
```suggestion
		const domainName = ownProps.cartDomainName ?? domain.name;
		const hasCartDomain = Boolean( ownProps.cartDomainName );
```

Reply:
Certainly makes sense ✌️